timemachine/fe/rbfe.py

```diff
@@ -135,18 +135,18 @@ def sample(initial_state, protocol):
 
     # a crude, and probably not great, guess on the decorrelation time
     n_steps = protocol.n_frames * protocol.steps_per_frame
-    all_nrgs, all_coords, all_boxes = ctxt.multiple_steps_U(
+    _, all_coords, all_boxes = ctxt.multiple_steps_U(
         lamb=initial_state.lamb,
         n_steps=n_steps,
-        lambda_windows=[initial_state.lamb],
```

Comment:
Avoids allocating an array on the GPU and avoids evaluating energies that we never use.